### DISCUSSION TITLE
Geo Data is not collected

### DISCUSSION THREAD

**Discussion by StephanSinn** [plausible/analytics#693] [Self-Hosted Support] [answered]
Dear Plausible Team,

first of all thank you for your Software and this great docker solution for self-hosting. **Compliments to you!** 

My plausible instance is currently not gathering any geo data, although I followed your describtions in detail and tried three different licence keys generated at MaxMind. I assume it should be the licence key for geoupdate > 3.1.1.
But still, I don't see any Geo Data collection in my dashboard. 

The download statistics from MaxMind shows one download for the first key. I read that the MaxMind geoupdate docker downloads every 7 days the database. Thus, the database retrieval seems to work. Why am I not seeing any Geo Data in my dashboard, what am I doing wrong?

Thanks in advance and best regards,
Stephan

**Comment by StephanSinn** [comment, marked as answer]
I don't know what happend, but now I gather the geo data. So I did not do so much wrong. Should be a little bit more patient...not used to it.

**Reply by ukutaht** [reply]
:) Glad it got fixed
